metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_usage_v2.py

```diff
@@ -66,8 +66,9 @@ class SnowflakeColumnReference(PermissiveModel):
 class SnowflakeObjectAccessEntry(PermissiveModel):
     columns: Optional[List[SnowflakeColumnReference]]
     objectDomain: str
-    objectId: int
     objectName: str
+    # Seems like it should never be null, but in practice have seen null objectIds
+    objectId: Optional[int]
```

Comment:
because this is a pydantic model, we don't need to reorder the fields - ideally we should make it match the sql query

Reply:
What does the order of the fields impact? I just changed it to group the optional fields together cause I thought it looked nicer. There's not actually a query order, because the whole object is a single json blob, `DIRECT_OBJECTS_ACCESSED`

ex:
```
            FROM
                (SELECT * from snowflake.account_usage.access_history) t,
                lateral flatten(input => t.DIRECT_OBJECTS_ACCESSED) r,
                lateral flatten(input => t.OBJECTS_MODIFIED) w
```

Reply:
Oh I see - alright then it's good as is